go/teams/teams.go

```diff
@@ -356,10 +360,15 @@ func (t *Team) isAdminOrOwner(m keybase1.UserVersion) (res bool, err error) {
 	return res, nil
 }
 
-func (t *Team) getDowngradedUsers(ms *memberSet) (uids []keybase1.UID, err error) {
+func (t *Team) getDowngradedUsers(ctx context.Context, ms *memberSet) (uids []keybase1.UID, err error) {
 
 	for _, member := range ms.None {
-		uids = append(uids, member.version.Uid)
+		// Load member first to check if their eldest_seqno has not changed.
+		// If it did, the member was nuked and we do not need to lease.
+		_, _, err := loadMember(ctx, t.G(), member.version, true)
```

Comment:
@mlsteele Can you take a look? Downgrade leases specifically check eldest_seqno and fail for nuked team_members: https://github.com/keybase/keybase/blob/master/web/downgrade_lease.iced#L200-L208

Reply:
Still something I should take a look at? Is the issue that the server is demanding you should have acquired a downgrade lease for a user who has been reset?

Reply:
If I got this right, it's a combination of these two - server specifically won't lease "outdated" users (lease endpoint wants latest eldest_seqno), but team handler wants these leases. I have this fixed in my server PR.

Sorry for not being clear though - I wanted to know if using `loadMember` here is fine. I can also change it to specifically check for `libkb.AccountResetError`.

Reply:
yeah loadMember seems good. Yes to explicitly checking for libkb.AccountResetError!